packages/graph/hash_graph/lib/graph/src/datastore/postgres/row_types.rs

```diff
@@ -3,18 +3,21 @@
 
 use sqlx::{types::Uuid, FromRow};
 
-use crate::types::{AccountId, DataType};
+use crate::types::AccountId;
+
+#[derive(Debug, FromRow)]
+pub struct ExistsRow {
+    pub exists: bool,
+}
```

Comment:
God this is strange 🙃 

Are we going to have to do this for basically any query we write? If so I'm starting to think the return values should live alongside the query definitions.

Reply:
Is it possible to use a 1-tuple for the deserialization instead? `contains_base_id` would look like
```rust
let (exists,) =
            sqlx::query_as(//....
```
and the type would be inferred by the return type

Reply:
This is **far** better, thank you! 5e39c68397677e2acd14c0dfa588c69d261f44f4